planetiler-custommap/src/test/java/com/onthegomap/planetiler/custommap/TestConfigurableUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.onthegomap.planetiler.custommap;
+
+import java.nio.file.Path;
+
+class TestConfigurableUtils {
+  static Path pathToTestResource(String resource) {
+    return resolve(Path.of("planetiler-custommap", "src", "test", "resources", "validSchema", resource));
+  }
+
+  static Path pathToTestInvalidResource(String resource) {
+    return resolve(Path.of("planetiler-custommap", "src", "test", "resources", "invalidSchema", resource));
+  }
+
+  static Path pathToSample(String resource) {
+    return resolve(Path.of("planetiler-custommap", "src", "main", "resources", "samples", resource));
+  }
+
+  private static Path resolve(Path pathFromRoot) {
+    Path cwd = Path.of("").toAbsolutePath();
+    return cwd.resolveSibling(pathFromRoot);
```

Comment:
Would `pathFromRoot.toAbsolutePath()` work here instead?

Reply:
No, because I needed to resolve from the project root, unless there's some better way to reference that point in the directory structure?